edflow/hooks/checkpoint_hooks/tf_checkpoint_hook.py

```diff
@@ -186,3 +186,55 @@ def wait(self):
 
     def before_epoch(self, ep):
         self.wait()
+
+
+class RestoreCurrentCheckpointHook(Hook):
```

Comment:
The only difference to `RestoreModelHook` is the `before_epoch` method? Then maybe just subclass and overwrite that method to avoid code repetition.

Reply:
should be fixed